### DISCUSSION TITLE
Improve sharing

### DISCUSSION THREAD

**Discussion by 255kb** [ehmorris/lunar-lander#19] [Ideas]
When clicking on share button it opens the navigator share interface, which on my Windows 11 computer doesn't offer much options. 
![image](https://user-images.githubusercontent.com/7489814/223370914-9c6a9896-f0f8-4ec8-9064-ad5b7a6cc2ff.png)

I suggest that you always copy the shareable text to the clipboard.

**Comment by ehmorris** [comment]
Thanks for the feedback. Right now I just ask the browser if it supports the navigator. If I can figure out a design with three buttons that's not obtrusive, I'll think about adding a copy button in there as well when there's support.
